website/blog/creating-cross-platform-push-to-talk-app.md

```diff
@@ -24,7 +24,7 @@ Here some pictures of the super place where our hackathon took place:
 
 ### Finding an Idea
 
-During our release phase, at the end of each sprint, everyone involved in the process meets up using [mumble](https://www.mumble.com/).
+During our release phase, at the end of each sprint, everyone involved in the process meets up using `https://www.mumble.com/`.
```

Comment:
www.mumble.com not reliable enough? oh well

Reply:
Yes it's reliable, but the linter just complains about `Error: unable to verify the first certificate`
So must have something weird ... i didn't investigate, since it's on a old unread blog post now